src/storage/mapping/ExtensionBasedMapper.ts

```diff
@@ -177,18 +74,17 @@ export class ExtensionBasedMapper implements FileIdentifierMapper {
    * @returns Content type of the file.
    */
   private getContentTypeFromExtension(path: string): string {
-    const extension = this.getExtension(path);
-    return (extension && this.types[extension.toLowerCase()]) || APPLICATION_OCTET_STREAM;
+    return this.types[this.getExtension(path).toLowerCase()] || APPLICATION_OCTET_STREAM;
```

Comment:
Could perhaps add this case in the constructor instead so this check is not needed anymore? After which this entire function can probably be removed and just put in `getContentTypeFromPath`.

Reply:
No; `APPLICATION_OCTET_STREAM` is not just for empty strings but also needed for unknown content types.

Reply:
Ah yea true, nevermind then. But this could still be merged with `getContentTypeFromPath` right since that is just a oneliner calling this function?

Reply:
Sync versus async. But yeah.